pkg/stream/http/connpool.go

```diff
@@ -345,6 +345,12 @@ func (ac *activeClient) OnResetStream(reason types.StreamResetReason) {
 				ac.client.ConnID())
 		}
 		ac.closeConn = true
+	} else if reason == types.StreamRemoteReset && !ac.closed {
```

Comment:
Good catch.  But whether the value of variable `reason` can also be set correctly in the exception. 
https://github.com/mosn/mosn/blob/625bfad6df6f760d3e4b48bd897ef7f7db749cdf/pkg/stream/http/stream.go#L292-L300